docs/overview/comparison.mdx

```diff
@@ -4,38 +4,36 @@ title: "Comparison"
 
-## Keep vs Alerta
+## Keep vs AIOps inside observability platforms (Splunk ITSI, Elastic AIOps, etc.)
 
-> alerta monitoring system is a tool used to consolidate and de-duplicate alerts from multiple sources for quick ‘at-a-glance’ visualisation. With just one system you can monitor alerts from many other monitoring tools on a single screen
-> [(alerta docs)](https://docs.alerta.io/).
+Keep differs because it's able to correlate alerts between different observability platforms.
 
-|                             | Keep                                                           | Alternative   |
-| --------------------------- | -------------------------------------------------------------- | ------------- |
-| Aggregation and correlation | ✅                                                             | ✅            |
-| Integrations                | ✅ (Both manually and automatically with Webhooks Integration) | ✅ (Manually) |
-| Alerts enrichment           | ✅                                                             | ❌            |
-| Open source                 | ✅                                                             | ✅            |
-| Workflow automation         | ✅                                                             | ❌            |
-| Managed version             | ✅                                                             | ❌            |
-| CI/CD integration           | ✅                                                             | ❌            |
-| Noise reduction             | ✅                                                             | ❌            |
+|                                       | Keep                                                           | Alternative                  |
+| ------------------------------------- | -------------------------------------------------------------- | ---------------------------- |
+| Aggregation of alerts from one platform | ✅                                                           | ✅                            |
+| .. from multiple platforms            | ✅                                                             | ❌                            |
+| Correlation of alerts between multiple sources | ✅                                                    | ❌                            |
+| Alerts enrichment                     | ✅                                                             | ❌                            |
+| Open source                           | ✅                                                             | ❌                            |
+| Workflow automation                   | ✅                                                             | ❌                            |
 
-## Keep vs Grafana
+## Keep vs dedicated AIOps solution (BigPanda, MoogSoft, etc.)
```

Comment:
Same as other comment but maybe repeating BigPanda and Moogsoft twice is redundant?
```suggestion
## Keep vs traditional AIOps platforms
```